containers/podman-desktop
# chore(test): add remote podman connection e2e test

### What does this PR do?
Add new e2e test spec file that will verify the podman-remote feature. Requires special environment setup prior the test run.

### Screenshot / video of UI



### What issues does this PR fix or reference?
#8632 


### How to test this PR?
Setup the env. in a specific way, see https://github.com/odockal/podman-desktop-qe/blob/main/docs/scenarios/podman-remote/podman-remote.md

We need to pull a specific image into podman machine we want to test: `ghcr.io/podmandesktop-ci/alpine-remote`
